Resources/Public/css/grid.7.6.css

```diff
@@ -0,0 +1,151 @@
+.flux-grid { width: 100%; table-layout: fixed}
+.flux-grid td { vertical-align: top; }
+.flux-grid-hidden { display: none; }
+.t3-page-ce-dragitem:hover .t3-page-ce-wrapper-new-ce { cursor: move; }
+
+.flux-grid tr:first-child:last-child td:first-child:last-child {
+	width: 100%;
+}
+.flux-grid tr:first-child:last-child td {
+	min-width: 174px;
+}
+@media (max-width: 750px) {
+	.flux-grid tr:first-child:last-child td {
+		min-width: 50%;
+	}
+}
+@media (max-width: 550px) {
+	.flux-grid tr:first-child:last-child td {
+		min-width: 100%;
+	}
+}
+.t3-page-lang-column .flux-grid td {
+	min-width: 100% !important;
+}
+.flux-grid-column > .t3-grid-cell {
+	margin-left: 10px;
+}
+.flux-grid-column:last-child > .t3-grid-cell {
+	margin-right: 10px;
+}
+
+/* copied from TYPO3 v8.6.0 */
```

Comment:
I would not override these CSS styles on LTS. Flux isn't supposed to homogenise the visual appearance on the two versions - our existing overrides are already vulnerable to changes and we can probably expect more of those! I'd like it if we aim to remove our overrides instead (grid.css is still needed though; and we need an additional patch to prevent height limiting and overflow-hidden on contains you are dragging elements into and which are being dragged).

And that should also mean we don't need the 7.6-specific CSS file - one less override to worry about ;)